client/logic/globals/multiplayer.gd

```diff
@@ -168,6 +168,7 @@ func _on_nakama_received_match_presence(new_presences: NakamaRTAPI.MatchPresence
 func _on_nakama_received_match_state(match_state: NakamaRTAPI.MatchData) -> void:
 	if match_state.op_code == OpCodes.MATCH_START:
 		random_seed = JSON.parse_string(match_state.data)["seed"]
+		_global.game_mode = GameConst.GameMode.NETWORK
```

Comment:
is it not dangerous to set this property globally? Or is reseted every time a match starts?

Reply:
It sets when choosing a map or start a multiplayer game. It is used to set the player to the right modes (Human, AI or Network)